### ISSUE TITLE
WARN deprecated rollup-plugin-babel@4.4.0

### ISSUE THREAD

**Issue by mudgen** [developit/microbundle#635]
When I installed microbundle I got this warning:

```
npm WARN deprecated rollup-plugin-babel@4.4.0: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-babel.
```

**Comment by mudgen** [comment]
It might be a good idea to update the rollup dependencies.

**Comment by developit** [comment]
This was already fixed in #616, will be released shortly.
